marketplace/plugins/s3/lib/operations.json

```diff
@@ -152,7 +152,7 @@
         "description": "Enter max keys",
         "width": "320px",
         "height": "36px",
-        "className": "codehinter-plugins col-6"
```

Comment:
We may have to look this in a case by case basis. I have updated for cases where it was found. Reason: As far as i understand, these classess where added for the plugin schema to have look according to old designs. Since we redesigned the layout, these classes are no longer valid